gradle/bootstrap-setup.gradle

```diff
@@ -21,8 +21,9 @@ configurations {
 			eachDependency {
 				if (requested.group == 'org.eclipse.xtext' || requested.group == 'org.eclipse.xtend')
 					useVersion(versions.xtext_bootstrap)
-				if (requested.group == 'com.google.inject' && requested.name == 'guice')
-					useVersion(versions.guice)
+// This should be obsolete now
```

Comment:
@cdietrich Is this correct?

Reply:
this code here is since the repo split. i have no idea what it is needed for. thus i kept it.